dir.props

```diff
@@ -41,6 +46,10 @@
     <TargetFrameworkIdentifier>.NETStandard</TargetFrameworkIdentifier>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <SkipBuildingTrimmingPackage Condition="'$(SkipBuildingTrimmingPackage)' == '' and '$(DotNetBuildFromSource)' == 'true'">true</SkipBuildingTrimmingPackage>
```

Comment:
While I realize this might be the only bad project there are plenty others that we should avoid building, so I would recommend a slightly different approach.  See my commit https://github.com/dotnet/standard/commit/7eac7d47620381b3023cf80c2e2556c5e54274e4 where I filtered more things that aren't needed for source-build.